src/tests/Microsoft.PowerFx.Core.Tests/DTypeTests.cs

```diff
@@ -1918,28 +1947,51 @@ public void TestCoercesTo()
             Assert.True(DType.ObjNull.CoercesTo(DType.Time));
             Assert.False(DType.Error.CoercesTo(DType.Time));
 
-            // Coercion to Attachment type
-            Assert.False(DType.Boolean.CoercesTo(AttachmentType));
-            Assert.False(DType.Number.CoercesTo(AttachmentType));
-            Assert.False(DType.Currency.CoercesTo(AttachmentType));
-            Assert.False(DType.Color.CoercesTo(AttachmentType));
-            Assert.False(DType.Guid.CoercesTo(AttachmentType));
-            Assert.False(DType.DateTime.CoercesTo(AttachmentType));
-            Assert.False(DType.Date.CoercesTo(AttachmentType));
-            Assert.False(DType.Time.CoercesTo(AttachmentType));
-            Assert.False(DType.String.CoercesTo(AttachmentType));
-            Assert.False(DType.Hyperlink.CoercesTo(AttachmentType));
-            Assert.False(DType.Image.CoercesTo(AttachmentType));
-            Assert.False(DType.PenImage.CoercesTo(AttachmentType));
-            Assert.False(DType.Media.CoercesTo(AttachmentType));
-            Assert.False(DType.Blob.CoercesTo(AttachmentType));
-            Assert.False(DType.EmptyTable.CoercesTo(AttachmentType));
-            Assert.False(DType.EmptyRecord.CoercesTo(AttachmentType));
-            Assert.True(AttachmentType.CoercesTo(AttachmentType));
-            Assert.False(DType.EmptyEnum.CoercesTo(AttachmentType));
-            Assert.False(DType.TryParse("%s[A:\"hello\"]", out type) && type.CoercesTo(AttachmentType));
-            Assert.True(DType.ObjNull.CoercesTo(AttachmentType));
-            Assert.False(DType.Error.CoercesTo(AttachmentType));
+            // Coercion to Attachment Table type
+            Assert.False(DType.Boolean.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Number.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Currency.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Color.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Guid.CoercesTo(AttachmentTableType));
+            Assert.False(DType.DateTime.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Date.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Time.CoercesTo(AttachmentTableType));
+            Assert.False(DType.String.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Hyperlink.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Image.CoercesTo(AttachmentTableType));
+            Assert.False(DType.PenImage.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Media.CoercesTo(AttachmentTableType));
+            Assert.False(DType.Blob.CoercesTo(AttachmentTableType));
+            Assert.False(DType.EmptyTable.CoercesTo(AttachmentTableType));
+            Assert.False(DType.EmptyRecord.CoercesTo(AttachmentTableType));
```

Comment:
>Assert.False(DType.EmptyRecord.CoercesTo(AttachmentTableType)); [](http://example.com/codeflow?start=12&length=63)

Trying to understand this scenario... If I have a SharePoint list where I can have an attachment (file), how can I do something like this?

    Collect(
      spList,
      {
        Title: "Foo",
        Attachment: {
          'Content-Type': "text/plain",
          Contents: "This is a simple file"
        }
      })

Reply:
This is not supported today: updating attachments is pretty much only possible through controls in PA Client
![image](https://user-images.githubusercontent.com/69215460/185205561-a3ab8273-de12-47e3-889b-5dc382b3ae1b.png)